### PR TITLE
add screenshot to readme and added instructions to test UI

### PR DESCRIPTION
**Pull request by madhuv-msft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,8 +41,22 @@ _dist_ folder contains a node.js file [server.js], which acts as server later.
...
 
+## How to test changes made in 'azpipelines-kubernetesUI'
+
+1. Build `azpipelines-kubernetesUI` repo, afetr making changes in the repo.
+
+2. Run following command, after replacing _\<src-location-of-zpipelines-kubernetesUI\>_ and _\<version\>_ with right values.
+
+    `npm install <src-location-of-zpipelines-kubernetesUI>\_bin\webAppPackage\azurepipelines-webapp-kube-summary-<version>.tgz`
+    
+3. After following command is done, navigate to **localhost:8095** using browser to check changes made.
```
Comment (savenkat) [inline README.md]: "After running the following command"

Comment (savenkat) [inline README.md]: "check the changes made"

Comment (madhuv-msft) [reply README.md]: #46 addressing these changes

Comment (madhuv-msft) [reply README.md]: #46 addressing these changes

**Pull request merged**